tests/ConfigurationTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+use Illuminate\Support\Facades\Config;
+use Spatie\Browsershot\Browsershot;
+use Spatie\LaravelPdf\Facades\Pdf;
+
+// Helper functions for testing Browsershot configuration
+function getPrivateProperty(object $object, string $property): mixed
+{
+    $reflection = new ReflectionClass($object);
+    $prop = $reflection->getProperty($property);
+    $prop->setAccessible(true);
+
+    return $prop->getValue($object);
+}
+
+function getBrowsershotOption(object $browsershot, string $key): mixed
+{
+    $options = getPrivateProperty($browsershot, 'additionalOptions');
+
+    return $options[$key] ?? null;
+}
+
+beforeEach(function () {
+    Config::set('laravel-pdf', []);
+});
+
+describe('Binary Path Configuration', function () {
```

Comment:
In our tests we never use `describe`. Remove this and just use `it` functions. Structure tests by adding multiple test files.